explorer/explorer.go

```diff
@@ -135,9 +140,50 @@ func (exp *explorerUI) rootWebsocket(w http.ResponseWriter, r *http.Request) {
 				exp.wsHub.HubRelay <- sigPingAndUserCount
 			}
 		}()
-
+		go func() {
+			for {
+				msg := &WebSocketMessage{}
+				websocket.JSON.Receive(ws, &msg)
```

Comment:
Here's where I'm concerned about someone sending a massive message.  I'm not sure how to limit this though.